### PR TITLE
Update ALFView interface documentation

### LINKED ISSUE #34824: ALFView - Update documentation and images of ALFView
**Description**
Update the interface documentation page for the ALFView interface. There are a few things I need to remember to add:
- [x] definition of terms such as the `Out of plane angle`
- [x] an explanation for how to delete regions on the InstrumentWidget. Maybe a tip within the interface itself would also be useful
- [ ] Add release notes for all changes

Update the release notes to give an overview of the key changes made to ALFView.

### PR DESCRIPTION
**Pull request by robertapplin**
**Description of work.**
This PR updates the interface documentation for ALFView, and adds release notes for recent feature changes made to the GUI.

The PR also adds a description for how to delete a shape when it is selected.

**To test:**
Build `docs-html`
Open the ALFView documentation page and read it. Make sure it makes sense/all links work/all images are displayed
Read the release notes

Fixes #34824

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/direct/ALFView.rst [hunk 1]
```diff
@@ -0,0 +1,123 @@
+.. _ALFView-ref:
+
+ALFView
+=======
+
+.. image::  ../../images/ALFView_Interface.png
```
Comment (DanielMurphy22) [inline docs/source/interfaces/direct/ALFView.rst:6]:
```suggestion
.. image::  ../../images/ALFView_Interface.PNG
```

**Review by DanielMurphy22 — CHANGES_REQUESTED** [review]
This PR should be merged tomorrow before code freeze! I wanted to approve, especially as the testing docs can be improved after testing, but the image in the main docs page should have capitals .PNG 

I'm off tomorrow so please get someone else to approve and merge!

**Review by DanielMurphy22 — DISMISSED** [review]
Quick change!

**Review by martyngigg — APPROVED** [review]

**Comment by martyngigg** [comment]
Unrelated test failures on documentation updates.

**Pull request merged**